megalinter/descriptors/kotlin.megalinter-descriptor.yml

```
@@ -40,3 +40,29 @@ linters:
       vim:
         - name: ale
           url: https://github.com/w0rp/ale
+  # DETEKT
+  - linter_name: detekt
+    can_output_sarif: true
+    linter_url: https://detekt.dev/
+    linter_repo: https://github.com/detekt/detekt
+    linter_rules_url: https://detekt.dev/docs/rules/comments
+    linter_banner_image_url: https://repository-images.githubusercontent.com/71729669/8b793230-5d85-4c36-b72e-6911dd7bf6d3
+    linter_megalinter_ref_url: https://detekt.dev/docs/intro
+    cli_lint_fix_arg_name: "--auto-correct"
+    cli_sarif_args:
+      - --report=sarif,output={{SARIF_OUTPUT_FILE}}
+    examples:
+      - "detekt-cli myfile.kt"
+      - "detekt-cli --auto-correct myfile.kt"
+    install:
+      dockerfile:
+        - |
+          RUN curl --retry 5 --retry-delay 5 -sSLO https://github.com/detekt/detekt/releases/download/v1.23.3/detekt-cli-1.23.3.zip &&
+            \ unzip detekt-cli-1.23.3.zip && \ mv detekt-cli-1.23.3/bin /usr/bin && \ chmod a+x /usr/bin/detekt-cli
```

The `\` is supposed to be a continuation character, to be able to continue to a new line without breaking.

It may be breaking in build, in that case, maybe this should work

```suggestion
          RUN curl --retry 5 --retry-delay 5 -sSLO https://github.com/detekt/detekt/releases/download/v1.23.3/detekt-cli-1.23.3.zip && \
              unzip detekt-cli-1.23.3.zip && \ mv detekt-cli-1.23.3/bin /usr/bin && \ chmod a+x /usr/bin/detekt-cli
```